GraphConsoleAppV3/GraphRequests.cs

```diff
@@ -0,0 +1,1534 @@
+        {
+            #region Get All Permissions
+
+            //*********************************************************************************************
+            // get all Permission Objects
+            //*********************************************************************************************
+            Console.WriteLine("\n Getting Permissions");
+            IPagedCollection<IOAuth2PermissionGrant> permissions = null;
+            try
+            {
+                permissions = await client.Oauth2PermissionGrants.ExecuteAsync();
+            }
+            catch (Exception e)
+            {
+                Program.WriteError("\nError Getting Permissions: {0} {1}", e.Message,
+                    e.InnerException != null ? e.InnerException.Message : "");
+            }
+            if (permissions != null)
+            {
+                do
+                {
+                    List<IOAuth2PermissionGrant> perms = permissions.CurrentPage.ToList();
+                    foreach (IOAuth2PermissionGrant perm in perms)
+                    {
+                        Console.WriteLine("Permission: {0}  Name: {1}", perm.ClientId, perm.Scope);
+                    }
+                    permissions = await permissions.GetNextPageAsync();
+                } while (permissions != null);
+            }
+
+            #endregion
+        }
+
+        #region Domain Operations
+
+        private static async Task PrintAllDomains(IActiveDirectoryClient client)
+        {
+            #region List all Domains
+
+            //*********************************************************************************************
+            // get all Domains
+            //*********************************************************************************************
+            Console.WriteLine("\n Getting Domains");
+            IPagedCollection<IDomain> domains = null;
+            try
+            {
+                domains = await client.Domains.ExecuteAsync();
+            }
+            catch (Exception e)
+            {
+                Program.WriteError("\nError Getting Domains: {0} {1}", e.Message,
+                    e.InnerException != null ? e.InnerException.Message : "");
+            }
+            while (domains != null)
+            {
+                List<IDomain> domainList = domains.CurrentPage.ToList();
+                foreach (IDomain domain in domainList)
+                {
+                    Console.WriteLine("Domain: {0}  Verified: {1}", domain.Name, domain.IsVerified);
+                }
+                domains = await domains.GetNextPageAsync();
+            }
+
+            #endregion
+        }
+
+        private static async Task<IDomain> CreateNewDomain(IActiveDirectoryClient client)
+        {
+            #region Create new Domain
+
+            IDomain newDomain = new Domain {Name = Helper.GetRandomString() + ".com"};
+            newDomain.IsVerified = true;
```

Comment:
>true [](start = 35, length = 4)

Is it possible to add verified domain without doing any additional steps? or should we be showing creation of non-verified domain?